lib/hexpm_web/controllers/dashboard/organization_controller.ex

```diff
@@ -213,8 +215,6 @@ defmodule HexpmWeb.Dashboard.OrganizationController do
         conn,
         organization,
         params,
-        # TODO: call fun.(customer_params, audit: audit_data(conn)) in
```

Comment:
I added this TODO because I found the `update_billing/4` method confusing. (We have a public function `update_billing/2` as the controller action handler.) And the call was getting more complex. 

I removed this TODO because I found it's not feasible to refactor this.

`Billing.create/2` and `Billing.update/3` need different params, so we can't just pass `Billing.create/2` or `Billing.update/3` into `update_billing/4` helper method.

I see two alternative ways here:
1. Add `organization` as a param to `Billing.create/2`, so it has the same arity as `Billing.update/3`
2. Inline the private method `update_billing/4`

Reply:
I think it is fine like is now.